tests/sample_data/transform.py

```diff
@@ -0,0 +1,364 @@
+import datetime
```

Comment:
this really should be called sample_data_generator.py, but that file exists and is a runtime thing.  I am trying to make this process easier in the future.